.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "codegen/botocore"]
```

Comment:
botocore uses its master branch for releases: we should use that instead of their default branch, `develop`.